ext/sockets/sockets.c

```diff
@@ -1026,7 +1026,7 @@ PHP_FUNCTION(socket_getsockname)
 #endif
 		case AF_INET:
 			sin = (struct sockaddr_in *) sa;
-			addr_string = inet_ntop(AF_INET, &sin->sin_addr, addrbuf, sizeof(addrbuf));
```

Comment:
I'm not totally certain that this change is better?

Reply:
it s more correctness, it s not factually fixing anything (unless something wrong in a libc side). It is more a recommendation to do so.